src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/templates/AppealSubmissionTemplate.java

```diff
@@ -48,15 +47,15 @@ public String getName() {
 
         fieldValues.put("hmcts", "[userImage:hmcts.png]");
         fieldValues.put("CREATED_DATE", caseDetails.getCreatedDate().format(DOCUMENT_DATE_FORMAT));
-        fieldValues.put("appealReferenceNumber", asylumCase.getAppealReferenceNumber().orElse(""));
-        fieldValues.put("legalRepReferenceNumber", asylumCase.getLegalRepReferenceNumber().orElse(""));
-        fieldValues.put("homeOfficeReferenceNumber", asylumCase.getHomeOfficeReferenceNumber().orElse(""));
-        fieldValues.put("homeOfficeDecisionDate", formatDateForRendering(asylumCase.getHomeOfficeDecisionDate().orElse("")));
-        fieldValues.put("appellantGivenNames", asylumCase.getAppellantGivenNames().orElse(""));
-        fieldValues.put("appellantFamilyName", asylumCase.getAppellantFamilyName().orElse(""));
-        fieldValues.put("appellantDateOfBirth", formatDateForRendering(asylumCase.getAppellantDateOfBirth().orElse("")));
+        fieldValues.put("appealReferenceNumber", asylumCase.read(APPEAL_REFERENCE_NUMBER, String.class).orElse(""));
```

Comment:
Is there any need to specify the type like String.class when reading a value?  I ask as I don't think it is being used and the code maps it to the correct type using the AsylumCaseDefinition enum.

Reply:
as discussed